### PR TITLE
create required resources

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:
- define the redirects resource

## security considerations
none

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -204,6 +204,23 @@ resources:
...
+    space: ((staging-cf-space))
+
+- name: deploy-stratos-redirects-production
+  type: cf
+  source:
+    api: ((production-cf-api-url))
+    username: ((production-cf-username))
+    password: ((production-cf-password))
+    organization: ((production-cf-organization))
+
```
Comment (tammersaleh) [inline ci/pipeline.yml]: shouldn't we have `space:` here?

Comment (tammersaleh) [reply ci/pipeline.yml]: ...also, since these aren't specific to the app, should we generalize the `deploy-stratos-*` resources instead?

Comment (tammersaleh) [reply ci/pipeline.yml]: (maybe I'm thinking about concourse resources incorrectly, tho)

Comment (bengerman13) [reply ci/pipeline.yml]: yes, space is required.
yes, these are functionally the same thing as the deploy-stratos-${ENV} resources

Comment (spgreenberg) [reply ci/pipeline.yml]: @tammersaleh You are correct in that the resource points to an org/space in a cf instance. It is not an app.

Comment (tammersaleh) [reply ci/pipeline.yml]: so maybe we should rename to `deploy-dev`, `deploy-staging`, `deploy-prod` and use for both apps?

Comment (tammersaleh) [reply ci/pipeline.yml]: (but maybe who cares 🤷‍♀)

Comment (bengerman13) [reply ci/pipeline.yml]: I used a different name, but, yeah, good idea

**Pull request merged**